web/docs/formats/xsv.md

```diff
@@ -14,10 +14,28 @@ Reads and writes lines with separated values.
 Parser:
 
 ```
-csv [--allow-comments] [--auto-expand] [--header <header>]
-ssv [--allow-comments] [--auto-expand] [--header <header>]
-tsv [--allow-comments] [--auto-expand] [--header <header>]
-xsv <field-sep> <list-sep> <null-value> [--allow-comments] [--auto-expand] [--header <header>]
+csv [--list-sep <list-sep>] [--null-value <null-value>]
+    [--allow-comments] [--auto-expand] [--header <header>]
+    [--merge] [--schema <schema>] [--selector <fieldname[:prefix]>]
+    [--expand-schema] [--raw] [--unnest-separator <separator>]
```

Comment:
We talked about potentially removing `--merge`, as there does not seem to be any use for this (under the assumption that we add `null` fields for non-available fields).